### PR TITLE
Issue 972 csv import

### PR DESCRIPTION
**Pull request by PawelJastrzebski**
https://github.com/quadratichq/quadratic/issues/972
- fix encoding issue 
https://github.com/quadratichq/quadratic/issues/970
- fix import csv with description before headers

### CODE REVIEW

File: quadratic-core/src/controller/import.rs [hunk 1]
```diff
@@ -18,25 +18,37 @@ impl GridController {
...
+                    let x: String = ut16.chars().filter(|&c| c.is_ascii()).collect();
+                    return self.import_csv(sheet_id, x.as_bytes(), file_name, insert_at, cursor);
+                }
+                file
+            }
+        };
 
         let mut reader = csv::ReaderBuilder::new()
             .has_headers(false)
+            .flexible(true)
```
Comment (davidkircos) [inline quadratic-core/src/controller/import.rs:34]: worth adding?

Comment (PawelJastrzebski) [reply quadratic-core/src/controller/import.rs:34]: Allows importing csv with variable row length (number of columns)

true
image
false
image

Not sure why test are passing with "false"

Comment (davidkircos) [reply quadratic-core/src/controller/import.rs:34]: That makes sense @ddimaria is looking into it!

**Comment by ddimaria** [comment]
Closing in favor of @PawelJastrzebski's other PR: https://github.com/quadratichq/quadratic/pull/1235

**Pull request closed without merging**